virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc

```diff
@@ -5,44 +5,24 @@ include::modules/virt-document-attributes.adoc[]
 :virt-importing-rhv-vm:
```

Comment:
It doesn't work because someone removed the attribute without telling me. I actually fixed this in another PR but this seems to have reverted. Will get rid of it.